### PR TITLE
Encourage using team ids

### LINKED ISSUE #454: Offer only valid team_ids
The current team_id-selectbox options within application/edit form, offer AWS account ids instead of team ids. However if one selects a team, that is not recognized by the teams API, the process of creating service users for the application will fail.

Please offer only valid team ids. Hint: The "owner" of the aws accounts could be used as possible options.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #454

### CODE REVIEW

File: client/lib/application/src/router.react.jsx [hunk 1]
```diff
@@ -136,7 +143,7 @@ class AppListHandler extends React.Component {
                     tabAccounts={tabAccounts}
                     selectedTab={selectedTab}
                     applicationsFetching={applicationsFetching}
-                    accounts={accounts}
+                    accounts={[...accountsAndTeams]}
```
Comment (ChristianLohmann) [inline client/lib/application/src/router.react.jsx]: redundant usage of spread dot operator

File: client/lib/common/src/router-utils.js [hunk 2]
```diff
@@ -1,9 +1,40 @@
-import * as UserGetter from 'common/src/data/user/user-getter';
 import REDUX from 'yourturn/src/redux';
+import {Pending} from 'common/src/fetch-result';
+import * as UserGetter from 'common/src/data/user/user-getter';
+import * as MagnificentGetter from 'common/src/data/magnificent/magnificent-getter';
+
+function requireAuth(state, team, magnificentActions) {
+    const permitted = MagnificentGetter.getAuth(state.magnificent, team);
+    // successful result
+    if (!(permitted instanceof Pending) && permitted !== null) {
```
Comment (ChristianLohmann) [inline client/lib/common/src/router-utils.js]: what if it's not `Pending` but something other?

**Comment by ChristianLohmann** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Comment by ChristianLohmann** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**